tests/container_disk_test.go

```diff
@@ -174,22 +146,36 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting multiple VMIs", func() {
 		Context("with ephemeral registry disk", func() {
 			It("[test_id:1465]should success", func() {
-				num := 5
-				vmis := make([]*v1.VirtualMachineInstance, 0, num)
-				objs := make([]runtime.Object, 0, num)
-				for i := 0; i < num; i++ {
-					vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
-					// FIXME if we give too much ram, the vmis really boot and eat all our memory (cache?)
-					vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1M")
-					obj := tests.RunVMI(vmi, 10)
+				const count = 5
+				var vmis []*v1.VirtualMachineInstance
+				for i := 0; i < count; i++ {
+					vmi, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), libvmi.NewCirros())
```

Comment:
This makes the test more expensive than before. Would you want to use less memory per VM (1M)?

Reply:
Yes, it makes the test more expensive. But the test wasn't touched since 2018 and in my opinion we should be able to run five proper VMIs in parallel on a single cluster. 

The original test said 

> // FIXME if we give too much ram, the vmis really boot and eat all our memory (cache?)

which I don't think should be an issue anymore.

Reply:
Cirros uses between 128Mi and 256Mi of memory, so worst case we spend 1280Mi of memory, which is easily exceeded by many other tests.

Reply:
But why should we waste 1GiB of RAM on this test? What is it good for?

Cirros requires this memory in order to boot. But in this test, we don't speak to the guest at all. I think that we make it too easy to write needlessly expensive tests. If a test does not check anything about the guest operating, it should not require more than 1M per VM.

Reply:
The memory was overridden with `vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1M")`, so I guess you can keep it in the new version as well.
Just add the proper builder that already exists (I think now it is only for the guest memory) or create a new one.